recipes/teensy_loader_cli/meta.yaml

```
@@ -0,0 +1,50 @@
+# Technically not 2.1.1, but it is not version 2.1, so ....
+{% set version = "2.1.1" %}
+
+package:
+  name: teensy_loader_cli
+  version: {{ version }}
+
+source:
+  git_url: https://github.com/PaulStoffregen/teensy_loader_cli.git
+  # Build off a recent master since 2019/08/28
+  git_rev: eb61aa1a482c2ad98916229e02bb8caae2464f48
+
+build:
+  number: 0
+  # For now, start with linux builds ony
+  skip: True
```

This is skipping everything
Should it be ?
```suggestion
  skip: True  # [win]
```